packages/builder/src/components/automation/AutomationBuilder/FlowChart/FlowChart.svelte

```diff
@@ -20,7 +14,8 @@
 
   let testDataModal
   let confirmDeleteDialog
-
+  let scrolling = false
+  let hasScrolled = false
```

Comment:
Looking at the usages of `hasScrolled`, I think it's redundant. We can just swap out any usages of it for `scrolling` and get the same behaviour so we may as well remove it.